app/models/cost_item.rb

```diff
@@ -1,5 +1,12 @@
 class CostItem < ActiveRecord::Base
   has_many :costs
 
+  validates :web_cost, inclusion: { in: %w(yes no) }
+  validates :name, :cost_group, presence: true
+
   scope :current, -> { where(current: true) }
+
+  def self.delivery_partners
+    Partners.delivery_partners
```

Comment:
Should this be all delivery partners? If so then maybe it's more obvious to remove this method because it makes it look like we're getting associated delivery_partners?